eotestanalysis_example.ipynb

```diff
@@ -0,0 +1,597 @@
Qs74odK/CEzDsRcYwksfaYLawodsUEW58WVV2oh5CWZVFvRMcIspyVP4sltlIH1lug1nGLHopwm+AFuPIOsZGImqmX5bqdtCKafXaSJostT3DYRa9lGGrQTjMWbixyHK9to7VSAvmOpFNTNFLEfbWn8OsIPVFvs4j373L0j3NUlmN9GNRDbKHDd2mDBu2dZgvSX2R7z7ZjFPDSAf2LM0upuiljODSNFnvGE1JqA+KWQIty3XagiUbaSDsPmF1MjvY0G2KyfIbWHh4IcuySDvF3JssDxdlscxGOsmyr2yWMYteSsn625YF2m0csm6ZzWKZjXRiyl02MYteyrCGmG3rT6OS9Xtp9dlIkmDgcquH2cMseinDb4xZtgIErT/2cKpvbHJRDpOBkRRW97KNWfRSgFmwRmOyaBzMmuDIctkNw0gOs+ilgLAFy3CYXBoHC5VjGOnBnqfZwhS9lGAd4UjCzvsmm3RSTIdhnYrD5GAkRTh6gT1Ps4UN3aYECxCcw4ay64diQ6tkHZOBkRQWM88wi17KsMkY0ZMxjPoky2FVDCMN5BsVsTaZHUzRSwFRMxODjTLLDdKsnPVPllfF8Ikqe5blYcRP+KXZnqvZoSqKnoh8W0TuFJGXRWS3iGwTkRUicpmIzAilnS8iWuBzXYHzXCwiy0WkW0R2isgyETmnQPo2EfmqiDwrIr0isklEbhCRhdUod7UIP/DDDTArDTKq3NYZNg5ZqcfFYvIw4iTKeGDP12xQLR+9jwOPAL8HNgGTgBOBJcD7ReREVX05lOcx4OaIYz0RdQIRuQL4JPAX4CfAeOBC4BYR+aiqXhlK3+pdz8nAQ8APgP2A84G/EpHTVfWB0otaO7Ku2NjbZmOS9XuZ9XZtpAdbWjKbVEvR61DV3vBGEbkc+ALweeBDod2PquqSYg4uIotwSt6LwPGqut3b/l3gYeAKEblVVdcEsn0Cp+TdCFygqkNenutxCubPRORIf3saCL5lZb0BWsfYGAQ7lvB3VrB2bSRNsN5ZHcweVRm6jVLyPG7wvg+u8BQf8L4v95U877xrgB8BrcB7/O3iarKf5zNBZU5VfwPcAxwOnFrhdVUd80lz2BtnYxCl5GTtnmatvEb6yPp601mn1pMx3uJ9r4zYN1dE/lFEvuB9H1XgOKd737+L2HdbKA3AgcD+wHOqurrIPKnAGqHDrB+Ngb24WF020oPVxWxS1Th6IvIpoB2YArwGOAWn5H0rIvkZ3ieYfxlwsaquDWybBOwDdKvqhojjPO99HxLYdqj3/VyeS43KE4mIPJxn12Fj5S2HLA5t5SMYFsBkYhiGURpB/9Dgb3ueZotqB0z+FLB34P/vgEtUdXNg2y7g6zg/uVXetqNwEzdOA+4UkaNVtcfbN8X73pnnnP72qYFt5eRJHdYYc5gc6hOzUDtMDkYSBOud1cHsUlVFT1VnA4jI3sAinCVvhYico6qPeGk2AV8JZb1bRM4E/gScALwXN0s2cVT1uKjtnqXv2FqcM6u+TFGYDNKPvZAUh1mmjTRgdTB71MRHT1VfUdWbgDOBGcC1ReQZAP7D+/v6wC7f+jaFaPztOyrMkxhBxc5WgojGZJJeSuk07D6aDIxkMANCdqnpZAxVfQl4CjhCRGYWkcUf4p0UOEYPsA5oF5E5EXn8Gb1Bf7xnve98PnhReRIjajaUzZCyB1IjYZ2MwyanGElgsRyzTRxLoM31vgeLSHui970qtP0u7/usiDxnh9KAi7e3FjhERBYUmScVWCeQwwIoNxZZtlaHVyXIqhyMZDBfvWxTsaInIoeIyKghUhFp8gImzwLuCwQ5PlZERp1XRN6AW2ED4Beh3Vd5318UkWmBPPOBDwN9wNX+dnU12c/zneD5RORc4HU4S+Mfiy9p7Qn68JgFxGgE/LocrtdZxeRgJInVvWxSjckYbwa+KSJ/AlYDW3Ezb08FDgA2Au8LpP8ecLCI3IdbzgzcrFs/pt2XVfW+4AlU9T4R+R5utYuVInIjbgm0C4DpwEdDq2L45zkHOA94QETuxMXWOx838/fv07QqBox+0zKnWUcwPIBRX5hi47A6bM+zpDCZG9VQ9O4ADsLFzDsGF7KkB+f/9nPgh6q6LZD+58DbgONxQ6jjgFdwq2hcqar3RJ1EVT8pIo/jLHjvB4Zw6+t+V1VvjUjfJyJnAJ8DLsJZCztxYV0uU9WnKix31QlOxrDG6QjKwuRSn9h9c2RdBlkvf1LYS4ZRsaKnqk8AHykh/U+Bn5Z5rqXA0hLS78KFcgmHc0kVYUXGfChy2LBf/RMO2ppVTOE1kiAczcHqYPaIYzKGMQZBK55NQBiJKb31jynqOaw+G3ET5RJkZItqr4xhlEk+S55ZAQyjvjHXAyMpLEi3AWbRSw353rKy3EBt5nH9Ew4rkkXCfrdZlYMRP+YOZIApekYdYEMP9Us48HeW751NtDKSwNqgYUO3KcMcZ3Pkk0GWZVKP2P1ymGXFSIpgG7T2mD1M0UsJFlolR9bL32hYnc5hcjDiJuinZ20xm9jQbQoIh5/I+lu/DTE0HnZPs112IzmCyp0pednEFL0UkG+2baH9jU4+vxJbJ7S+CIdVyWpHY+FljCTxn5v27MwmpuilhHAIBuscHeE3Uesw64tg55LlTibLZTfSg82Czyam6KWEqIkHWW6Ipsw1BkHFPKv31F5OjDRgk9qyiyl6KcIaXo6g83CWFd56xyx6DqvHRtJY/csupuilmKwqfuHAslmVQ6ORVWu11WPDMJLEFL2UEJxtm6VOMB9ZH+5rNLI+bGTWaSNJgu4DVgezhyl6KSHY+LLWCYaxB5HRiJivnpEkthRfdjFFL2VkPayKjy171jhkdcg2iNVnI0mC0RzMlSB7mKKXMsJhVrKOPZQagyyv+hJV5izKwUgWe7nILqbopQxrjCOxlULqn6jYkFnC6q+RNBaXNduYopcygpMQrIMwjMbB2rORFMFwVUb2aEn6Aoz8ZPnNK/wGakO49Y1NNsp
```

Comment:
This looks pretty good until ~150000 and then the variance shoots up in the last bin.  Is that saturation, or something else?